spec/visual/samples/cmhg

```diff
@@ -0,0 +1,58 @@
+;
+; CDDL HEADER START
+;
+; The contents of this file are subject to the terms of the
+; Common Development and Distribution License (the "Licence").
+; You may not use this file except in compliance with the Licence.
+;
+; You can obtain a copy of the licence at
+; cddl/RiscOS/Sources/HWSupport/SD/SDIODriver/LICENCE.
+; See the Licence for the specific language governing permissions
+; and limitations under the Licence.
+;
+; When distributing Covered Code, include this CDDL HEADER in each
+; file and include the Licence file. If applicable, add the
+; following below this CDDL HEADER, with the fields enclosed by
+; brackets "[]" replaced with your own identifying information:
+; Portions Copyright [yyyy] [name of copyright owner]
+;
+; CDDL HEADER END
+;
+; Copyright 2012 Ben Avison.  All rights reserved.
+; Portions Copyright 2013 Jeffrey Lee.
+; Use is subject to license terms.
+;
+
+#include "Global/Services.h"
+#include "VersionNum"
+
+initialisation-code:	module_initialise
+
+finalisation-code:	module_finalise
+
+; Service_Hardware
+service-call-handler:	module_service Service_Hardware,
+				       Service_ModulePostInit
+
+title-string:		SDIODriver
+
+help-string:		SDIODriver Module_MajorVersion_CMHG Module_MinorVersion_CMHG
+
+command-keyword-table:	module_command
+			SDIODevices(min-args:0, max-args:0, international:, invalid-syntax:"SSDIDEV", help-text:"HSDIDEV"),
+			SDIOSlots(min-args:0, max-args:0, international:, invalid-syntax:"SSDISLT", help-text:"HSDISLT")
+
+swi-chunk-base-number:	0x59000
```

Comment:
Could you add a test of non-hexadecimal digits to the visual sample?

Reply:
There were some, but they were all 0, so not very interesting. This particular sample was just copied directly from a real project, but it doesn't really matter for our purposes if we change the numbers!